advance_check_void/models/account_payment.py

```diff
@@ -0,0 +1,391 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT
```

Comment:
You should not need this.
Instead is [fields.Date methods](https://www.odoo.com/documentation/14.0/developer/reference/addons/orm.html#odoo.fields.Date).
```suggestion
```